src/components/navigationLoader/navigationMenuSectionItem.tsx

```diff
@@ -0,0 +1,13 @@
+import * as React from 'react';
```

Comment:
Same here. Could be renamed to `NavigationMenuSectionItemLoading`, or simply `NavigationItemLoading`? or keep the name and add the required code to make this a real navigation item to render a real link with a title.

File names should start with a capital letter.